modules/microshift-security-context-constraints-alert-eval.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * microshift_running_apps/microshift-authentication.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations
+
+The `PodSecurityViolation` alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by reviewing the Kubernetes API server audit logs. This procedure uses the `must-gather` tool to gather the audit logs and then searches for the `pod-security.kubernetes.io/audit-violations` annotation.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To gather the audit logs, enter the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather -- /usr/bin/gather_audit_logs
```

Comment:
Sorry for the delay in responding here, @kelbrown20 

We should have instructions for collecting an sos report elsewhere in our troubleshooting docs already, since that was work we did in 4.13.

MicroShift does not have alerts in the same what that OpenShift does, so I'm not sure how to reproduce these steps. Where did the original version come from? What is the goal of this section of the docs?

Reply:
No worries! Okay, I just found the [procedure in the docs for collecting an sos report ](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.13/html/support/microshift-sos-report). I'll remove those details from this PR. 👍 

I based this new doc set off of the [Pod security section in the OCP docs ](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html). From my understating of the Jira story, the main information that needs to be added to the MicroShift docs are the details on Synchronization with pod security standards and Controlling pod security admission synchronization, which is what I'm adding in this PR. Is that correct, or was there a part of that I was missing?

Reply:
@pacevedom does an sos report for MicroShift include an audit log that could be used to identify pod security violations in a way similar to what's described below?

I don't see `pod-security.kubernetes.io/audit-violations` in the audit log files on my system where I know I have generated PSA violations. I do see messages with `pod-security.kubernetes.io/enforce-policy`. @stlaz since MicroShift is enforcing restricted PSA already, are we going to need the same sort of investigation techniques that OCP users need?

Reply:
I dug into the where the PSA audits are kept. The logs are stored on the microshift host under /var/log/kube-apiserver/audit.log in a json format. I was able to force a PSA violation and see it captured there.  I don't believe that sos is including that file when it's gathering though.